sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/SpannerIOReadChangeStreamTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
```

Comment:
Could we also add an IT where we set the credentials in this directory? 

https://github.com/apache/beam/blob/59e9ee9e02e9f95d4cdaee065529594ae86a72d4/sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/it/SpannerChangeStreamTransactionBoundariesIT.java#L61

Reply:
It seems a bit tricky to set up a real service account/credential to be used in an integration test. For now, I'm able to run pipelines with my own service account and verify the behavior.